doc/examples/README.md

```diff
@@ -5,3 +5,11 @@ This folder contains a collection of more specific examples using flame, special
 To run this examples after cloning, first run `flutter create .` inside to generate the necessary files, as per instructed by the other flutter commands.
 
 After that, you should be able to `flutter run` just fine.
+
+In order to simplify the understanding of each examples, they are all missing the following folders:
+
+```
+android, ios, web, macos, test
+```
+
+So that we can focus on the `lib` and (if applicable) `assets` folder. Please add that to the gitignore of your new example.
```

Comment:
Is it possible to split these changes to a different PR? 

I am afraid this onde became unnecessarily big and it is doing a lot more than what is in the title/description.